packages/template-block/src/TemplateBlock.tsx

```diff
@@ -74,6 +80,26 @@ export const TemplateBlock = ({ appBridge }: BlockProps): ReactElement => {
         ? `${borderWidth_blockCard} ${borderStyle_blockCard} ${toRgbaString(borderColor_blockCard)}`
         : 'none';
 
+    const getCardPadding = () => {
```

Comment:
Can these functions move to a helper file?
getCardPadding and getLayoutClasses

Reply:
Refactored 👌
Now it's much cleaner indeed.